tests/installation/agama.pm

```diff
@@ -84,9 +84,21 @@ sub agama_define_user_screen
     send_key 'ret';
 }
 
+sub upload_agama_logs {
+    return if (get_var('NOLOGS'));
+    select_console("root-console");
+    # stores logs in /tmp/agma-logs.tar.gz
+    script_run('agama logs store');
+    upload_logs('/tmp/agama-logs.tar.gz');
+}
+
+sub get_agama_install_console_tty {
+    # get_x11_console_tty would otherwise autodetermine 2
+    return 7;
```

Comment:
Will this tty be changed in future? ：）

Reply:
https://github.com/openSUSE/agama/pull/1592/files#diff-3484c3867e4b1f3c4834bf10baead2e74c824da3895151d758d8676193a5da63 hopefully in next version